functions/New-DbaLogShippingPrimaryDatabase.ps1

```diff
@@ -0,0 +1,262 @@
+The name if the prefix, the function will add the name of the database.
+
+.PARAMETER BackupRetentionMinutes
+Is the length of time, in minutes, to retain the log backup file in the backup directory on the primary server. 
+
+.PARAMETER BackupThresshold
+Is the length of time, in minutes, after the last backup before a threshold_alert error is raised. The default is 60 minutes. 
+
+.PARAMETER Compression
+Specifies whether a log shipping configuration uses backup compression. This parameter is supported only in SQL Server 2008 Enterprise (or a later version). 
+If the parameter is not set and the version of SQL Server is equal or higher than SQL Server 2008 the default of the server will be retrieved.
+
+.PARAMETER AlertEnabled
+Specifies whether an alert will be raised when backup_threshold is exceeded.
+
+.PARAMETER ThressholdAlertMinutes
+Is the alert to be raised when the backup threshold is exceeded. The default is 14,420. 
+
+.PARAMETER HistoryRetentionMinutes
+Is the length of time in minutes in which the history will be retained. The default is 14,420.
+
+.PARAMETER UseDatabaseSuffix
+Is used to change several parameter values to include the name of the databases. 
+Parameters that will be changed with this parameter are BackupDirectory, BackupShare
+
+.PARAMETER Force
+When set the original settings will be overwriten.
+
+.PARAMETER Silent
+Whether the silent switch was set in the calling function.
+If true, it will write errors, if any, but not write to the screen without explicit override using -Debug or -Verbose.
+If false, it will print a warning if in wrning mode. It will also be willing to write a message to the screen, if the level is within the range configured for that.
+
+.PARAMETER WhatIf
+Shows what would happen if the command were executed 
+
+.NOTES 
+Original Author: Sander Stad (@sqlstad, sqlstad.nl)
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+.LINK
+https://dbatools.io/New-DbaLogShippingPrimaryDatabase
+
+.EXAMPLE   
+
+.EXAMPLE   
+#>
+
+    [CmdletBinding(SupportsShouldProcess = $true)]
+
+    param (
+		[parameter(Mandatory = $true)][ValidateNotNullOrEmpty()]
+		[object]$SqlServer,
+		[System.Management.Automation.PSCredential]$SqlCredential,
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [string[]]$Database,
```

Comment:
Function does not handle multiple databases, should also set this as `[object]$Database`.